pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -132,6 +134,7 @@ type Config struct {
 		TokenURL           string   `gcfg:"token-url"`
 		TokenBody          string   `gcfg:"token-body"`
 		ProjectID          string   `gcfg:"project-id"`
```

Comment:
Add comment `// ProjectID can either be the numeric or string-based unique identifier that starts with [a-z]`